src/relations/Relation.ts

```diff
@@ -66,7 +66,10 @@ export type ManyOf<ModelName extends KeyType> = Relation<
   any
 >
 
-export type RelationsMap = Record<string, Relation<any, any, any>>
+export type RelationsList = Array<{
+  path: string[]
```

Comment:
What do you think if we name this property `propertyPath` to be aligned with how this is represented in the rest of the code?

Reply:
I agree, done.